ubuntu24.04/Dockerfile

```diff
@@ -64,8 +64,11 @@ RUN echo "TARGETARCH=$TARGETARCH"
 ADD install.sh /tmp
 
 # Fetch GPG keys for CUDA repo
-RUN apt-key del 7fa2af80 && OS_ARCH=${TARGETARCH/amd64/x86_64} && OS_ARCH=${OS_ARCH/arm64/sbsa} && \
-    apt-key adv --fetch-keys "https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/${OS_ARCH}/3bf863cc.pub"
+RUN apt-key del 7fa2af80 3bf863cc && \
```

Comment:
Do we know if these GPG keys are already added to the container image?

If not, we could just remove it and add them when it becomes necessary

Reply:
3bf863cc is included. 7fa2af80 is the old one and it is not in the ubuntu 24.04 image. I can omit 7fa2af80 from this command.